code/modules/admin/tabs/admin_tab.dm

```diff
@@ -872,3 +872,17 @@
 
 	SSticker.mode.toggleable_flags ^= MODE_BYPASS_JOE
 	message_admins("[src] has [MODE_HAS_TOGGLEABLE_FLAG(MODE_BYPASS_JOE) ? "allowed players to bypass (except whitelist)" : "prevented players from bypassing"] Working Joe spawn conditions.")
+
+/client/proc/toggle_joe_respawns()
+	set name = "Toggle Working Joe Respawns"
+	set category = "Admin.Flags"
+
+	if(!admin_holder || !check_rights(R_EVENT, FALSE))
```

Comment:
```suggestion
	if(!admin_holder || !check_rights(R_EVENT, TRUE))
```
I don't see much reason why the invalid rights message should be suppressed? You can do the same for `toggle_bypass_joe_restriction` above.